images/core.proto

```diff
@@ -53,6 +54,7 @@ message task_core_entry {
 	//optional int32		tty_pgrp	= 17;
 
 	optional bool			child_subreaper	= 18;
+	optional uint64			blk_sigset_mips = 16[(criu).hex = true];
```

Comment:
Let's rename it into `blk_sigset_extended` - in case there're other architectures with a big sigset.